apps/vr-tests-web-components/package.json

```diff
@@ -18,7 +18,7 @@
     "html-react-parser": "4.0.0",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32344&commitId=d36231d89c82d6aa7d8ceb2b8b08c8f0e41c7239&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|14| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|4918| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>